src/test/java/com/linecorp/armeria/server/docs/DocServiceTest.java

```diff
@@ -52,17 +53,24 @@
     @Override
     protected void configureServer(ServerBuilder sb) {
         final ThriftService helloService = new ThriftService(HELLO_SERVICE_HANDLER);
-        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class));
-        final ThriftService cassandraService = new ThriftService(mock(Cassandra.AsyncIface.class));
+        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class),
+                                                           SerializationFormat.THRIFT_BINARY,
+                                                           EnumSet.of(SerializationFormat.THRIFT_BINARY));
```

Comment:
This doesn't look user-friendly. `ThriftService` could automatically include the default serialization format to the allowed serialization format list?